sourcecoop/egg-source-coop.json

```diff
@@ -0,0 +1,152 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
+    "meta": {
+        "version": "PTDL_v2",
+        "update_url": null
+    },
+    "exported_at": "2024-05-14T22:54:17-07:00",
+    "name": "SourceCoop",
+    "author": "red_thirten@yahoo.com",
+    "description": "SourceCoop is a cooperative server-side-only mod for multi-player Source Engine games (like Black Mesa), that enables players to play together on single-player campaigns or maps. This Egg currently supports official Black Mesa maps only, but can be easily modified to support more (see Egg README for details).",
+    "features": null,
```

Comment:
no steam disk features?

Reply:
Ahh, sorry, forgot about these hidden features. I'll add that tomorrow. I wonder if Pelican will include them in the UI?

Reply:
I hope so, I have asked them to add them.

Reply:
Added 👍